expfmt/openmetrics_create_test.go

```diff
@@ -397,14 +403,16 @@ request_duration_microseconds_count 2693
 				Metric: []*dto.Metric{
 					&dto.Metric{
 						Counter: &dto.Counter{
-							Value: proto.Float64(42),
+							Value:            proto.Float64(42),
+							CreatedTimestamp: openMetricsTimestamp,
```

Comment:
Looks like you added created timestamps to all counters and summary tests. Since created timestamp is optional, should we maybe keep a variation with created and not created?